lib/cc/engine/analyzers/php/main.rb

```diff
@@ -20,7 +20,7 @@ class Main < CC::Engine::Analyzers::Base
           private
 
           def process_file(path)
-            code = File.read(path)
+            code = File.binread(path)
```

Comment:
It's such a minor change I thought I'd keep it in since it's probably not worth its own PR.